### PR TITLE
Fix/remove doc filters

### LINKED ISSUE #1809: Poster UX: Remove Markdown to Chain proposal type
Change: Remove "Post Markdown to Chain" proposal option.

Rationale: Currently, there is no meaningful difference in use case / utility for posting something to chain vs IPFS, so let’s default to posting all docs to IPFS.

### LINKED ISSUE #1810: Poster UX: Change filters to "Ratified" and "All"
- [x] Remove "IPFS" filter option
- [x] Remove "Chain" filter option

Rationale: The current IPFS/Chain filter is not meaningful in helping users filter through docs.

### PR DESCRIPTION
**Pull request by brossetti1**
## GitHub Issue

https://github.com/HausDAO/daohaus-app/issues/1810
https://github.com/HausDAO/daohaus-app/issues/1809

## Changes

- [x] Remove "IPFS" filter option
- [x] Remove "Chain" filter option
- [X] Remove Option for posting to chain since there is no difference currently between use cases for IPFS and on chain

I have opted for commenting out the option for on chain posting as if use case can change later we can simply uncomment the option for on chain posting.

## Packages Added

N/A

## Checks

Before making your PR, please check the following:

- [X] Critical lint errors are resolved
- [X] App runs and builds locally

## Details

Screen Shot 2022-05-25 at 12 52 09 PM
Screen Shot 2022-05-25 at 1 01 28 PM

### CODE REVIEW

File: src/components/proposalActions.jsx [hunk 1]
```diff
@@ -518,7 +518,8 @@ const ProposalVote = ({
                   }
                 />
               ) : (
-                quorumNeeded && (
+                quorumNeeded &&
+                !proposal.isMinion && (
```
Comment (plor) [inline src/components/proposalActions.jsx]: This got mixed in from your other PR. It doesn't belong in this PR.

File: src/data/formLegos/posterForms.js [hunk 2]
```diff
@@ -65,21 +65,21 @@ export const POSTER_FORMS = {
...
+  //   required: ['posterData.title', 'posterData.content'],
+  //   fields: [
+  //     [
+  //       FIELD.POST_TITLE,
+  //       POSTER_DESCRIPTION,
+  //       FIELD.MD_EDITOR,
+  //       FIELD.POSTER_ENCODER,
+  //     ],
+  //   ],
+  // },
```
Comment (plor) [inline src/data/formLegos/posterForms.js]: I would suggest just deleting rather than commenting out.

File: src/data/txLegos/posterBoostTX.js [hunk 3]
```diff
@@ -38,14 +38,14 @@ export const POSTER_BOOST_TX = {
...
+  // POSTER_MD: {
+  //   contract: CONTRACTS.POSTER,
+  //   name: 'post',
+  //   onTxHash: ACTIONS.PROPOSAL,
+  //   poll: 'subgraph',
+  //   display: 'Posting...',
+  //   errMsg: 'Error Posting Doc',
+  //   successMsg: 'Posted Doc!',
+  //   gatherArgs: ['.values.encoded', POSTER_TAGS.MEMBER],
+  // },
```
Comment (plor) [inline src/data/txLegos/posterBoostTX.js]: Same, if we need any of this back we can pull it from the history.

**Review by plor — CHANGES_REQUESTED** [review]

File: src/components/proposalActions.jsx (hunk 1 above)
Comment (brossetti1) [reply src/components/proposalActions.jsx]: whoops! sorry

**Pull request merged**